libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Microsoft.Bot.Builder.Dialogs.Adaptive.csproj

```diff
@@ -45,11 +45,14 @@
     <PackageReference Include="Microsoft.Bot.Builder.LanguageGeneration" Condition=" '$(IsBuildServer)' != '' " Version="$(PreviewPackageVersion)" />
     <PackageReference Include="Microsoft.Bot.Expressions" Condition=" '$(IsBuildServer)' == '' " Version="4.6.0-local" />
     <PackageReference Include="Microsoft.Bot.Expressions" Condition=" '$(IsBuildServer)' != '' " Version="$(PreviewPackageVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="2.1.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.UserSecrets" Version="2.1.0" />
     <PackageReference Include="Microsoft.Recognizers.Text" Version="1.2.9" />
     <PackageReference Include="Microsoft.Recognizers.Text.Choice" Version="1.2.9" />
     <PackageReference Include="Microsoft.Recognizers.Text.DateTime" Version="1.2.9" />
     <PackageReference Include="Microsoft.Recognizers.Text.Number" Version="1.2.9" />
     <PackageReference Include="Microsoft.Recognizers.Text.Sequence" Version="1.2.9" />
+    <PackageReference Include="RichardSzalay.MockHttp" Version="6.0.0" />
```

Comment:
> <PackageReference Include="RichardSzalay.MockHttp" Version="6.0.0" /> [](start = 3, length = 70)

this shouldn't be here.

Reply:
It should be as long as Adaptive tests include MockLuis.  I'll file a work item.

---
In reply to: [360172545](https://github.com/microsoft/botbuilder-dotnet/pull/2936#discussion_r360172545) [](ancestors = 360172545)